rules/library.bzl

```diff
@@ -991,6 +996,7 @@ def apple_library(name, library_tools = {}, export_private_headers = True, names
         pch = pch,
         data = [] if swift_sources else [module_data],
         tags = tags_manual,
+        defines = defines + objc_defines,
```

Comment:
If `foo=bar` is declared in `defines` and `foor=bar1` in `objc_defines`, do you know whether the compiler will take the later or will it spit error? 🙏

Reply:
You can quickly test that in the [dummy project](https://github.com/thiagohmcruz/swift-objc-libraries-defines-issue) I linked in the issue. It would just trigger a warning:
```sh
<command line>:4:9: warning: 'foo' macro redefined [-Wmacro-redefined]
#define foo bar1
        ^
<command line>:3:9: note: previous definition is here
#define foo bar
```
and the context would define what value is used